src/userInterface/components/CommandItem/CommandItem.tsx

```diff
@@ -1,12 +1,26 @@
 import React from 'react';
 import Button from "antd/es/button";
 import styles from "./CommandItem.module.scss";
+import Spin from 'antd/es/spin';
 
-export const CommandItem: React.FC<{ className?: string, onClick?: () => void, icon?: string }> = (props) => {
-    return (<Button type="primary" className={`${styles.commandBtn} ${props.className}`}
-                    icon={props.icon}
-                    size="large"
-                    onClick={props.onClick}>
-            {props.children}
-    </Button>);
+export const CommandItem: React.FC<{ className?: string, onClick?: () => void, icon?: string, loading?: boolean }> = (props) => {
+    const buttonProps = props.loading ?
+        {
+            disabled: true
+        } :
+        {
+            icon: props.icon,
+            onClick: props.onClick
+        }
+
+    return (
+        <Button
+            type="primary"
+            className={`${styles.commandBtn} ${props.className}`}
+            size="large"
+            {...buttonProps}
```

Comment:
`disabled={props.loading}`
 How about this change. As I can remember spread props is not allowed in EsLint